source/core/src/main/com/deco2800/game/entities/factories/PlayerFactory.java

```diff
@@ -30,7 +34,7 @@
    * Create a player entity.
    * @return entity
    */
-  public static Entity createPlayer() {
+  public static Entity createPlayer(GdxGame game) {
```

Comment:
Does this really need to be passed to the player? Could there be a listener for when the player dies?